protocol/0057-TRAN-transfers.md

```diff
@@ -5,63 +5,101 @@ These transfers are not to be confused with the internal concept of transfers wh
 
 The amount paid at the end of each epoch is calculated using the following formula:
+
 ```
 amount = start amount x factor ^ (current epoch - start epoch)
 ``` 
 
-If not enough funds are present in the source account at the time a transfer is initied by the network, the whole recurring transfer is cancelled.
-
+If insufficient funds are present in the source account at the time a transfer is initied by the network, the whole recurring transfer is cancelled.
 If the `amount` is less than `transfer.minTransferQuantumMultiple x quantum` then the recurring transfer is cancelled. 
 
+
 ## Recurring transfers to reward accounts
-To be able to dispatch rewards to reward pools of the given markets pro-rata to the contribution of the reward metric (e.g. received maker fees) in the market to the total of the measured asset recurring transfers support auto dispatch in the following way:
 
-When transferring to a reward account, it is possible to define the reward metric, the reward metric asset, and a subset of markets. If the markets are not defined - it is taken as all the markets that settle in the reward metric asset. At the end of the epoch when the transfer is about to be distributed, it first calculates the contribution of each market (either out of all the markets that settle in the reward metric asset or only the ones in scope of the transfer) to the total reward metric and then distributes the transfer to the corresponding accounts of the markets pro-rata. For example: a transfer is defined as follows:
+Read this section alongside the [rewards](./0056-REWA-rewards_overview.md) specification.
+
+To be able to dispatch rewards to reward pools of multiple markets pro-rata to the contribution of the reward metric (e.g. received maker fees) in the market vs the total of the measured metric across all in scope markets, recurring transfers support auto dispatch in the following way:
+
+- When transferring to a reward account, it is possible to define the reward metric, the reward metric asset, and a subset of reward markets. If the reward markets are not defined - it is taken as all the markets that settle in the reward metric asset which must be specified when creating the recurring transfer. 
+
+- At the end of the epoch when the transfer is about to be distributed, it first calculates the contribution of each market (either out of all the markets that settle in the reward metric asset or only the ones in scope of the transfer) to the total reward metric and then distributes the transfer to the corresponding accounts of the markets pro-rata. 
+
+Where the reward metric type is "market creation rewards", it is important that no market creator will receive more than one market creation reward paid in the same asset from the same source account (reward funder). 
+Therefore: 
+
+- A list of [market, source account, reward asset] combinations that have already been rewarded is maintained.
```

Comment:
I think this should be a mapping from market -> [[markets_in_scope], source account, reward asset], otherwise I don't understand it. Where the market on the LHS is the market whose proposer has been paid, and the RHS is the combo for which they've been paid for. 

As written or as I understand it at least, in the following example, a, b will only be paid by the earlier of [t1, t2] which is not what I thought we wanted nor what is implemented. 
 
t1: funder = zohar, markets_in_scope = [a,b,c], payout_asset = vega
t2:funder = zohar, markets_in_scope = [a,b], payout_asset = vega

Reply:
Oops I missed that update, fixed